contracts/contracts/Coordinator.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+/**
+* @title CoordinatorV1
+* @notice Coordination layer for DKG-TDec
+*/
+contract CoordinatorV1 {
+
+    uint8 public constant DKG_SIZE = 8;
+
+    event NewRitual(uint32 indexed ritualID, address[] nodes);
+    event TranscriptPosted(uint32 indexed ritualID, address indexed node, bytes32 transcriptDigest);
+    event ConfirmationPosted(uint32 indexed ritualID, address indexed node, address[] confirmedNodes);
+
+    enum RitualState {
+        NON_INITIATED,
+        WAITING_FOR_TRANSCRIPTS,
+        WAITING_FOR_CONFIRMATIONS,
+        ENDED
+    }
+
+    // TODO: Find better name
```

Comment:
could you explain what a Performance is?

Reply:
`Performance` would be the information associated to an individual participant of a `Ritual`. In this case, transcript data (technically, just a commitment of this data) and confirmations (i.e., what other nodes in the Ritual have validated them)

Reply:
Hmm, maybe something like `Contribution` or `Commitment`?

Reply:
I hope we can agree on `Participant` 😄